apps/language_models/scripts/vicuna.py

```diff
@@ -983,7 +1024,7 @@ def __init__(
         low_device_memory=False,
         weight_group_size=128,
         download_vmfb=False,
-        cache_vicunas=False,
+        cache_vicunas=True,
```

Comment:
Not sure it particularly matters here, but I'd prefer we leave this as false as it generates a lot of GB worth of files on disk

Reply:
Oops, thought I'd taken that out before committing. The command line flag wasn't working for me, so I hacked that in so I didn't have to generate the first and second vicuna data on every run.

Reply:
Should be fixed with https://github.com/nod-ai/SHARK/pull/1702/commits/0cb54b305a0e1af62c43185ba68aa90798c665be